valter-junior/projetoIP
# Adicionando múltiplos inimigos

**What I did**

- I Atualized the main with multiples enemies 
- Added a new enemy class Inimigo2
- I've put a temporary start and game over screen for tomorrow checkpoint

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout main`
   2. `git pull`
   3. `git checkout feature/novo`
   4. `git pull origin feature/novo`
   
3. Test the things that you think are worth testing, even the ones that are not in this description
   - test if the is working how expected; 
   - test if it's working on the main;
